lowfi_simulation/src/main/native/include/MotorEncoderSimulator.h

```diff
@@ -0,0 +1,34 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#pragma once
+
+#include <memory>
+
+#include "EncoderSimulator.h"
+#include "MotorSimulator.h"
+
+namespace frc {
+namespace sim {
+namespace lowfi {
+
+class MotorEncoderSimulator {
+ public:
+  MotorEncoderSimulator(const std::shared_ptr<MotorSimulator>& motorController,
+                        const std::shared_ptr<EncoderSimulator>& encoder);
+  ~MotorEncoderSimulator();
```

Comment:
If this is supposed to be virtual, it should be `virtual ~MotorEncoderSimulator() = default;` since the destructor is empty (derived classes would then use `~DerivedClass() override;` for a non-empty impl or `~DerivedClass() override = default;` for an empty default impl). If not, the destructor can be omitted completely.

Reply:
Dumb, I'm used to the IDE generating that for me but my eclipse (or how I imported the gradle project) isn't set up great so I did it by hand and forgot it.